engine/apps/alerts/models/channel_filter.py

```diff
@@ -116,7 +116,10 @@ def is_satisfying(self, raw_request_data, title, message=None):
         return self.is_default or self.check_filter(json.dumps(raw_request_data)) or self.check_filter(str(title))
 
     def check_filter(self, value):
-        return re.search(self.filtering_term, value)
+        try:
+            return re.search(self.filtering_term, value)
+        except re.error:
+            logger.error(f"channel_filter={self.id} failed to parse regex={self.filtering_term}")
```

Comment:
Probably a good idea to add `return None` or `return False` here, to make it more verbose